babylon-to-espree/toAST.js

```diff
@@ -94,7 +94,11 @@ var astTransformVisitor = {
     if (path.isRegExpLiteral()) {
       node.type = "Literal";
       node.raw = node.extra.raw;
-      node.value = {};
+      try {
+        node.value = new RegExp(node.pattern, node.flags);
+      } catch (err) {
+        node.value = null;
```

Comment:
Could you give a scenario in tests that handles when the `value` is `null`? Also, perhaps we should have it default to `{}` as it was before? Otherwise this could be a breaking change right?